content/product/iac.md

```diff
@@ -0,0 +1,241 @@
+                    final var cluster = new Cluster("eks-cluster");
+                    ctx.export("kubeconfig", cluster.kubeconfig());
+                  }
+                }
+            - title: Pulumi.yaml
+              language: yaml
+              code: |
+                resources:
+                  eks-cluster:
+                    type: eks:Cluster
+                outputs:
+                  kubeconfig: ${cluster.kubeconfig}
+          button:
+            text: "Learn more about Pulumi Packages"
+            link: "/product/packages/"
+          features:
+              - title: Native cloud providers
+                description: |
+                    Full API coverage for AWS, Azure, Google Cloud, and Kubernetes with same-day updates.
+              - title: Crosswalk for AWS
+                description: |
+                    Adopt well-architected best practices for your infrastructure easily with the [Crosswalk library](/docs/clouds/aws/guides/).
+              - title: Cloud Native support
+                description: |
+                    Use a single workflow to manage both [Kubernetes](/kubernetes/) resources and infrastructure.
+
+        - title: "Deliver infrastructure through software delivery pipelines"
+          sub_title: "CI/CD Integrations"
+          description: |
+            Version, review, test, and deploy infrastructure code through the same tools and processes used for your application code.
+          image: "/images/product/pulumi-cicd.png"
+          button:
+            text: "Learn more about CI/CD Integrations"
+            link: "/docs/using-pulumi/continuous-delivery/"
+          features:
+              - title: Version and review
+                description: |
+                    Manage infrastructure code in Git and approve changes through pull requests.
+              - title: Shift left
+                description: |
+                    Get rapid feedback on your code with fast [unit tests](/docs/using-pulumi/testing/unit/), and run [integration tests](/docs/using-pulumi/testing/integration/) against ephemeral infrastructure.
+              - title: Continuous delivery
+                description: |
+                    [Integrate your CI/CD provider](/docs/using-pulumi/continuous-delivery/) with Pulumi or use GitOps to [manage Kubernetes clusters](/docs/using-pulumi/continuous-delivery/pulumi-kubernetes-operator/).
+
+stats:
+    title: Open source. Enterprise ready.
+    description: |
+        Pulumi’s Universal Infrastructure as Code CLI and SDK is an [open-source project](https://github.com/pulumi/) that’s supported
+        by an active community. We maintain a [public roadmap](/blog/relaunching-pulumis-public-roadmap/) and welcome feedback and contributions.
+    community:
+        number: "10,000s"
+        description: of community members
+    company:
+        number: "1,000s"
+        description: of companies
+    integration:
+        number: "150+"
```

Comment:
I think there is more than this now. i think we should do 170+ since https://www.pulumi.com/registry/ is showing 178